### PR TITLE
Ensure #setUp is classified in running

### PR DESCRIPTION
**Pull request by jecisc**
We currently have a release test that #setUp should be classified in #running but the current MethodClassifier put it in initialization. This update the behavior by adding a list of methods we want to see in specific protocols + add a test

### CODE REVIEW

**Review by MarcusDenker — APPROVED** [review]

File: src/Tool-Base/MethodClassifier.class.st [hunk 1]
```diff
@@ -49,53 +59,54 @@ MethodClassifier class >> buildPragmaMapping [
...
+		                 at: 'requires' put: 'testing';
+		                 at: 'reset' put: 'initialization';
+		                 at: 'set' put: 'initialization';
+		                 at: 'should' put: 'asserting';
+		                 at: 'shouldnt' put: 'asserting';
+		                 at: 'signal' put: 'signalling';
+		                 at: 'sort' put: 'sorting';
+		                 at: 'test' put: 'tests';
+		                 at: 'total' put: 'accessing';
+		                 at: 'version' put: 'versions';
```
Comment (astares) [inline src/Tool-Base/MethodClassifier.class.st:106]: @jecisc why not "versioning" instead of "versions" if we follow the Someth**ing** pattern

Comment (jecisc) [reply src/Tool-Base/MethodClassifier.class.st:106]: This was already here, I just changed the formatting. It's possible there are more things to improve. I just wanted to align the release tests and the method classifier

**Comment by astares** [comment]
@jecisc And I do not understand: "setUp is in running and MethodClassifier put it in running too" what the issue is?

Is it about symbol (#running) vs. string ("running") or what?

**Comment by jecisc** [comment]
We have a release test that says that "setUp" method should be in "running" protocol. 

But the method categorizer of Pharo was putting "setUp" in the "initialization" protocol by default because the prefix map says that things starting by "set" are going in the initialization protocol

**Comment by jecisc** [comment]
Oh, I see that I wrote too fast in my original message. It should be: 

We currently have a release test that #setUp should be classified in #running but the current MethodClassifier put it in initialization

Sorry for that

**Comment by astares** [comment]
@jecisc Ah - OK. Thanks for the clarification!

**Pull request merged**